src/ci_providers/provider_azurepipelines.js

```diff
@@ -57,9 +57,10 @@ function _getSHA(inputs) {
 
   if (_getPR(inputs)) {
     const mergeCommitRegex = /^[a-z0-9]{40} [a-z0-9]{40}$/
-    const mergeCommitMessage = childProcess.execSync(
-      'git show --no-patch --format="%P"',
-    )
+    const mergeCommitMessage = childProcess
+      .spawnSync('git', ['show', '--no-patch', '--format="%P"'])
+      .stdout.toString()
```

Comment:
```suggestion
      .execFileSync('git', ['show', '--no-patch', '--format="%P"'])
      .toString()
```

:blush:

Reply:
@ra80533 this appears to have caused a failure in tests. It's because we didn't update tests, right?

Reply:
If the tests are hooking up into spawnSync, yes.

Reply:
Yah, every thing is mocked, as the results of git in ci are non-deterministic